src/Components/Form/Form.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import ApiRequest from '../ApiRequest/ApiRequest';
+import './Form.scss';
```

Comment:
do not import scss files inside components !
Create one index.scss file, import it in index.js file
And other partials, components files import inside thath index.scss file using sass imports